pymc/distributions/logprob.py

```diff
@@ -90,21 +93,23 @@ def _get_scaling(total_size, shape, ndim):
                 "number of scalings is bigger that ndim, got %r" % total_size
             )
         elif (len(begin) + len(end)) == 0:
-            return floatX(1)
+            coef = 1.0
```

Comment:
Are you sure about bot returning immediately here like it was doing before? Do we even have tests for this function?

Reply:
Yes, I am sure this change made the difference for the code, afair it was float32/64 related issue.